multiboot2/Cargo.toml

```diff
@@ -32,7 +32,10 @@ repository = "https://github.com/rust-osdev/multiboot2"
 documentation = "https://docs.rs/multiboot2"
 
 [features]
-default = []
+# by default, builder is included
+default = ["builder"]
+std = []
```

Comment:
I think, but I'm not sure right now, that we do not need a std module but only an alloc module. Which code from the standard library do you need for the builder which doesn't work with no_std+alloc?

Reply:
Yeah, please name the module `alloc`. I'll fix this in the header crate as well. It makes no sense to depend on `std`

Reply:
You can leave it as it is. I'll refactor this in #135